CHANGELOG.md

```diff
@@ -157,6 +157,9 @@ Adding a new version? You'll need three changes:
   to `/status/ready`. Gateways will be considered ready only after an initial
   configuration is applied by the controller.
   [#4368](https://github.com/Kong/kubernetes-ingress-controller/pull/4368
+- Moved protocol overriding to `kongstate.Override` step for expression based
+  routes, instead of adding expressions to match protocols.
```

Comment:
I'm not sure this is an entry that a typical user would understand. `kongstate.Override` is an implementation detail isn't it?

What is the user facing change here? Can we rephrase it to have a user facing meaning?

Reply:
User will see the `protocols` moved from expressions to `protocols` field of translated Kong route.

Reply:
updated.